app/javascript/components/shared_components/utilities/RoundPlaceholder.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Placeholder } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+
+export default function RoundPlaceholder({ radius, className }) {
+  return (
+    <Placeholder animation="glow">
+      <Placeholder style={{ height: radius, width: radius, borderRadius: '50%' }} className={className} bg="secondary" />
+    </Placeholder>
+  );
```

Comment:
Creating components with manually specified heights and widths passed in as a prop is bad practice. We should be matching the style used in other components (logo and placeholder)

Reply:
Agreed. We could replace the width/radius by a fixed class. It needs to be done for Avatars as well.